pilot/pkg/config/kube/crd/conversion.go

```diff
@@ -111,6 +111,9 @@ func ResourceName(s string) string {
 
 // ResourceGroup generates the k8s API group for each schema.
 func ResourceGroup(schema *model.ProtoSchema) string {
+	if schema.GroupDomain != "" {
+		return schema.Group + schema.GroupDomain
```

Comment:
This is missing a `.` in between values -- `model.IstioAPIGroupDomain` is set to `.istio.io`, whereas we set our domain to `maistra.io`. I think we should inject the `.` here instead of having to put it in the `ProtoSchema`s

Reply:
Good catch